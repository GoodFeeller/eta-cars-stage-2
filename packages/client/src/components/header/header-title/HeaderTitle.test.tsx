import { render, screen } from '@testing-library/react'
import HeaderTitle from './HeaderTitle.tsx'
test('HeaderTitle snapshot test', () => {
  render(<HeaderTitle />)
  const title = screen.queryAllByRole('div')
  expect(title).toMatchSnapshot()
})
