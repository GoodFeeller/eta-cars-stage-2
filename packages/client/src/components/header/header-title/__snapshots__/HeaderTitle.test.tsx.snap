// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTitle snapshot test 1`] = `[]`;
